Doc/library/argparse.rst

```diff
@@ -1868,6 +1868,10 @@ Argument groups
     The function exists on the API by accident through inheritance and
     will be removed in the future.
 
+   .. versionchanged:: 3.14
+    Passing prefix_chars_ to :meth:`add_argument_group`
```

Comment:
Also, are there alternative patterns that we'd suggest here? Using a custom action?  Perhaps worth mentioning.

Reply:
It seems there is a wrong indentation.

There is no alternative pattern, because, as far as we know, there is no use case for this. It never worked, it was just an implementation artifact. We only add a deprecation warning instead making it an error because we want to be nice in case if somebody used it for the reasons unknown to us.

If there is a good use case for this, we would rather document and fix this feature.

Reply:
Fair - makes sense.